docs/devel/roadmap.md

```diff
@@ -0,0 +1,57 @@
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Containers|P1|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Pods|P1|
+|Monitoring/troubleshooting   |Surface errors/warnings for all resources shown|P1|
+|Manage apps/resources        |YAML edit for resources displayed|P1|
+|Manage apps/resources        |Delete support for resources displayed|P1|
+|Create apps                  |Deploy form: migrate to Deployment objects|P1|
+|Multi cluster (Ubernetes)    |Support multiple clusters in one UI|P1|
+|Security/IAM                 |Make the UI securely accessible from outside of a cluster|P1|
+|Security/IAM                 |Make the UI act on behalf of a user|P1|
+|Security/IAM                 |IAM: don't crash/work correctly when IAM is enabled|P1|
+|Nonfunctional                |Analytics|P1|
+|Nonfunctional                |"Send feedback"/"Report bug" capability|P1|
+|View cluster resources       |Show interconnections between resources: related resources, parents, children|P2|
+|Create apps                  |Deploy form: autosuggest from a container registry (GCR, docker hub, etc.)|P2|
+|Create apps                  |Deploy form: implement missing options (e.g., volumes, etc.)|P2|
+|View cluster resources       |Show logs for controller resources (e.g., logs for all Pods of a Replica Set)|P2|
+|View cluster resources       |Handle Containers - list and basic details view|P2|
+|View cluster resources       |Handle Secrets - list and basic details view|P2|
+|View cluster resources       |Handle Namespaces - list and basic details view|P2|
+|View cluster resources       |Handle Ingress - list and basic details view|P2|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Services|P2|
+|Monitoring/troubleshooting   |Logs viewer: show logs for O(days) time period|P2|
+|Monitoring/troubleshooting   |Logs viewer: handle large log files - O(GiB) in size|P2|
+|Monitoring/troubleshooting   |Events viewer: handle large logs streams - O(1k) events|P2|
+|Multi cluster (Ubernetes)    |Native support of Ubernetes control plane: Dashboard can talk to it|P2|
+|Security/IAM                 |IAM: support IAM as first class citzen (e.g., disable a button when cannot delete a resource)|P2|
+|Nonfunctional                |Internationalization|P2|
+|View cluster resources       |<other resources> - list and basic details view|P3|
+|Monitoring/troubleshooting   |Sparklines/metrics/graphs for Replica Sets/Controllers/Deployments|P3|
+|Monitoring/troubleshooting   |Logs viewer: aggregate logs across pods/containers/controllers|P3|
+|Monitoring/troubleshooting   |Events viewer: aggregate events across pods/containers/controllers|P3|
+|Manage apps/resources        |Custom edit views on resources|P3|
+|Manage apps/resources        |Custom actions on specific resources (e.g., rollback on deployments)|P3|
+|Create apps                  |Plugin integration with deployment tools (such as helm, deployment manager or kpm)|P3|
```

Comment:
I would rank it higher, as it is interesting to us. But I wouldn't want to spend much effort on it. Rather some integration point. Priority depends a bit as the discussion is moving forward.